lib/util/fs/index.spec.ts

```diff
@@ -204,6 +206,51 @@ describe('util/fs/index', () => {
     });
   });
 
+  describe('readLocalSynmlink', () => {
+    it('reads symlink', async () => {
+      await withDir(
+        async (localDir) => {
+          GlobalConfig.set({
+            localDir: localDir.path,
+          });
+          await writeLocalFile('test/test.txt', '');
+          await fs.symlink(
+            resolve(localDir.path, 'test/test.txt'),
+            resolve(localDir.path, 'test/test')
+          );
+
+          const result = await readLocalSymlink('test/test');
+          expect(result).not.toBeNull();
+        },
+        {
+          unsafeCleanup: true,
+        }
+      );
+    });
+
+    it('return null when link not exists', async () => {
+      await withDir(
+        async (localDir) => {
+          GlobalConfig.set({
+            localDir: localDir.path,
+          });
+          await writeLocalFile('test/test.txt', '');
+          await fs.symlink(
+            resolve(localDir.path, 'test/test.txt'),
+            resolve(localDir.path, 'test/test')
+          );
+
+          const notExistsResult = await readLocalSymlink('test/not-exists');
+
+          expect(notExistsResult).toBeNull();
+        },
+        {
+          unsafeCleanup: true,
+        }
+      );
+    });
+  });
```

Comment:
`localDir` is automatically provided for each test:

```suggestion
  describe('readLocalSymlink', () => {
    it('reads symlink', async () => {
      await writeLocalFile('test/test.txt', '');
      await fs.symlink(
        resolve(localDir, 'test/test.txt'),
        resolve(localDir, 'test/test')
      );

      const result = await readLocalSymlink('test/test');

      expect(result).not.toBeNull();
    });

    it('return null when link not exists', async () => {
      await writeLocalFile('test/test.txt', '');
      await fs.symlink(
        resolve(localDir, 'test/test.txt'),
        resolve(localDir, 'test/test')
      );

      const notExistsResult = await readLocalSymlink('test/not-exists');

      expect(notExistsResult).toBeNull();
    });
  });
```